python/TestHarness/TestHarness.py

```diff
@@ -789,6 +803,16 @@ def parseCLArgs(self, argv):
         tmp_str = getattr(self.options, key)
         setattr(self.options, key, value[0])
 
+    # If attempting to test only failed_tests, open the .failed_tests file and create a list object
+    # otherwise, open the failed_tests file object for writing (clobber).
+    self.test_list = []
+    failed_tests_file = os.path.join(os.getcwd(), '.failed_tests')
```

Comment:
This works, because run_tests is changing to the directory when instancing the TestHarness:

```python
# Set the current working directory to the directory where this script is located
os.chdir(os.path.abspath(os.path.dirname(sys.argv[0])))
```

The ClusterLauncher, and PBS also depend on the above assumption. As does the TestHarness os.walk routine:

```python
      else:
        self.options.processingPBS = False
        self.base_dir = os.getcwd()
        for dirpath, dirnames, filenames in os.walk(self.base_dir, followlinks=True):
```

Its probably not the best way, but I was just following suite.